src/args.rs

```diff
@@ -479,10 +480,10 @@ impl<'a> ArgMatches<'a> {
         Ok(self.str_pattern(s))
     }
 
-    /// Converts a &str pattern to a String pattern, including word
+    /// Converts a &str pattern to a String pattern, including line/word
     /// boundaries or escapes if applicable.
     fn str_pattern(&self, pat: &str) -> String {
-        let s = self.word_pattern(self.literal_pattern(pat.to_string()));
+        let s = self.line_pattern(self.word_pattern(self.literal_pattern(pat.to_string())));
```

Comment:
Yeah I'd definitely like to stick to a 79 column limit (inclusive). I'd probably just write it like this:

```rust
let litpat = self.literal_pattern(pat.to_string());
let s = self.line_pattern(self.word_pattern(litpat));
```

But if you want to make a new method, that's cool too!